File: submissions/sm_110_mihir/week_3/day_3/session_2/average_divisible_while.js
function avg_div(limit,divisor){
    var num=1,  element=[],index=0, sum=0
    //check divisor is greater than zero
    if (divisor > 0){
        while(num<=limit){
            // if number is divisible, add it to the array.
            if(num%divisor==0){
                element[index++]=num
            }
            num++
        }
        console.log('divisible elements:',element)
        // traversing array for sum of elements
        index=0
        while(index<element.length){
            sum+=element[index]
            index++
        }
        console.log('Average=',(sum/element.length))
    }
}

avg_div(10,2)

File: submissions/sm_112_soumik/week_13/day_4/session_1/average_divisible_while.py
limit = int(input("Enter a limit :"))
div = int(input("Enter a num to check div :"))
i = 0
to_sum = 0
while i < limit:
    if i % div == 0:
        to_sum += i
    i = i + 1
print(to_sum // limit)
